Makefile

```diff
@@ -18,6 +18,7 @@ build:
 	mvn package -DskipTests=true
 	cp ./target/$(artifact_name)-$(version).jar ./$(artifact_name).jar
 
+
```

Comment:
I would remove this new, empty blank line.

Reply:
removed